runtime/cmake/platform/toolcfg/gnu.cmake

```diff
@@ -20,7 +20,8 @@
 # SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 ################################################################################
 
-list(APPEND OMR_PLATFORM_COMPILE_OPTIONS -O3 -g -fstack-protector)
+list(APPEND OMR_PLATFORM_COMPILE_OPTIONS -O3 -g -fstack-protector -Wreturn-type)
+list(APPEND OMR_PLATFORM_C_COMPILE_OPTIONS -Wimplicit)
```

Comment:
This leads to numerous warnings:
```
cc1plus: warning: command line option ‘-Wimplicit’ is valid for C/ObjC but not for C++
```
see [runtime/compiler/CMakeLists.txt](https://github.com/eclipse/openj9/blob/master/runtime/compiler/CMakeLists.txt#L228).

Reply:
do you have the updated pr? There shoud be the jit fix

Reply:
Yes, you're right. I forgot to absorb that change.